### ISSUE TITLE
Add proper article content to krayate & intrudr

### ISSUE THREAD

**Issue by iandioch** [iandioch/cpssd.net#3]

**Comment by devoxel** [comment]
See #26

**Comment by byxor** [comment]
Perhaps include 2nd year projects too.

**Comment by byxor** [comment]
Intrudr has now been removed. Whether or not someone wants to write about it is up to them, but it's unlikely that any of the original members plan to.

**Comment by iandioch** [comment]
Nor me - I don't know anything about it. We can let it be lost to the sands of time.

**Comment by devoxel** [comment]
I do agree with including second year projects. 

Also maybe getting those dang first years / forth years to add their first year projects, though that's unlikely. :rage1: :rage2: :rage3: :rage4:

**Comment by iandioch** [comment]
I will prob add a bit about Dipla sooner or later. Probably later.

**Comment by byxor** [comment]
Should the posts be about...?
A) The project
B) Making the project
C) Both

and

Should the posts be...?
A) Honest
B) Dishonest but flashy and much good PR with very good the marketing

**Comment by iandioch** [comment]
Up to you - I'll just decide case by case. The site can mirror or link to personal blog posts if you think they're interesting and representative of or related to the course somehow. Eg: 
- "Ross and I went to this hackathon"
- "I built this cool thing"
- "We did a group project for a year, here's what I learned"
- "This guy on our course won this marathon, congratulations"

A mix would be great. Re: CPSSD projects:
- My thinking was that it would be useful for interviews & CVs and whatever to have somewhere to link to when you mention Dipla, etc. In that case, a description of the project and its features, and perhaps some evaluation of it, would be most useful.
- A promo for Dipla doesn't make interesting reading; a page saying "here's what's wrong with how we made dipla" would also be cool, and could be perhaps more valuable in some situations.

If you have any personal blog or project or whatever and you want some free SEO, feel free to link it. I think somewhere I have a list of articles to write about some programming competitions we took part in too.

**Comment by iandioch** [comment]
TL;DR: Any one of [A, B, C], plus anything else interesting, always [X].
